tools/rng_test/README.md

```diff
@@ -0,0 +1,151 @@
+# RNG(Random Number Generator) duplication testing tool
+
+This tool can be used to find duplicate random numbers generated by auth_generate_random() function.
+
+It repeats the following two operations.
+
+1. Call auth_generate_random() (executed by 'security_api' TASH command)
+2. Reboot
+
+
+
+### Prerequisites
+
+* Python 2.7 & 3.x
+
+* Install 'pySerial'
+
+  ```
+  ~$ pip install pyserial
+  ```
+
+  
+
+### How to use
+
+1. Enable Security API configurations in menuconfig.
+
+   * Application Configuration - Examples - Security Test 
+     * [*] Security API Test
+   * Security Framework 
+     * [*] Security API
+     * [*] Authentication
+     * [*] Cryptography
+     * [*] Key manager
+     * [*] Secure storage
+
+   
+
+2. Change settings if you need
```

Comment:
1st one should be done in a device side
2nd and 3rd should be done in Ubuntu.
How about saying that clearly?

Reply:
I modified the explanation in more detail.